bundles/org.openhab.binding.satel/README.md

```diff
@@ -381,34 +404,29 @@ then
     ]
 end
 
-rule "Send event log start"
+rule "Send event log"
 when
     Item Alarms changed to ON
 then
-    eventLogCounter = 0
-    eventLogMsgBody = ""
-    EVENT_LOG_PREV.postUpdate(NULL)
-    EVENT_LOG_IDX.sendCommand(-1)
-end
-
-rule "Send event log next"
-when
-    Item EVENT_LOG_PREV changed
-then
-    if (EVENT_LOG_PREV.state == NULL) {
-        
-    } else if (eventLogCounter == 30) {
-        sendMail("my@address.net", "Alarm system log", eventLogMsgBody)
-        // prevent initiating reading when index item is restored during OH startup
-        EVENT_LOG_IDX.postUpdate(NULL)
-    } else {
-        eventLogMsgBody += "\n" + (EVENT_LOG_TIME.state as DateTimeType).format("%1$tF %1$tR") + ": " + EVENT_LOG_DESCR.state
-        if (EVENT_LOG_DET.state != NULL && EVENT_LOG_DET.state != "") {
-            eventLogMsgBody += " - " + EVENT_LOG_DET.state
-        }
-        eventLogCounter += 1
-        EVENT_LOG_IDX.sendCommand(EVENT_LOG_PREV.state)
+    val actions = getActions("satel", "satel:event-log:home:EventLog")
+    if (null === actions) {
+        logInfo("EventLog", "Actions not found, check thing ID")
+        return
     }
+    logInfo("EventLog", "Start")
+    var msgBody = ""
+    var eventIdx = -1
+    (1..10).forEach[
+        val eventRec = actions.readEvent(eventIdx)
+        val details = eventRec.get("details")
+        msgBody += "\n" + String::format("%1$tF %1$tR", eventRec.get("timestamp")) + ": " + eventRec.get("description")
+        if (details != NULL && details != "") {
+             msgBody += " - " + details
+        }
+        eventIdx = eventRec.get("prev_index")
+    ]
+    logInfo("EventLog", "End")
+    getActions("mail","mail:smtp:local").sendMail("you@email.net", "Event log", msgBody)
```

Comment:
Maybe add a note seomwhere that this requires the mail binding.

Reply:
This is just an example. The purpose of the action is to read single record from the event log. It does not specify what one should do with the result.

Reply:
True. I was just thinking that someone installs `satel`, adds this example and runs into an error. Maybe a comment in the line above this one like `sending notifications via mail requires the mail binding` or something like that would be good.

Reply:
Done.